crates/release_plz/src/commands/init_github.rs

```diff
@@ -223,6 +255,68 @@ impl ReleaseMetadataBuilder for NoopReleaseMetadataBuilder {
     }
 }
 
+mod gh {
```

Comment:
Copied this in from the previous module as I didn't want to create a whole init folder and gh.rs in there for this much code. Not 100% wedded to the idea here. Suggest something more preferable if you want it.

Reply:
I prefer files instead of `mod` 👍